com.unity.ml-agents/Runtime/Sensors/CameraSensor.cs

```diff
@@ -75,6 +75,17 @@ public int[] GetObservationShape()
             return m_Shape;
         }
 
+        /// <summary>
+        /// Accessor for the dimension properties of a camera sensor. A camera sensor
+        /// Has translational equivariance along width and hight and no property along
+        /// the channels dimension.
+        /// </summary>
+        /// <returns></returns>
+        public DimensionProperty[] GetDimensionProperties()
+        {
+            return new DimensionProperty[] { DimensionProperty.TranslationalEquivariance, DimensionProperty.TranslationalEquivariance, DimensionProperty.None };
```

Comment:
Cache these (or make it a static member of CameraSensor or IDimensionPropertiesSensor) so that we don't allocate each step.